frontend/next.config.js

```diff
@@ -111,6 +111,13 @@ let app = withPlausibleProxy({
             },
         })
 
+        // This lets us import yaml files, and have their contents
+        // be baked into the output bundle as a static string
+        config.module.rules.push({
+            "test": /\.yaml/,
+            "type": "asset/source",
+        })
```

Comment:
I feel like it might be better to `fetch` the file as text and use `import.meta.url`

Reply:
I initially expected the `yaml` import to parse the data like importing `json` does

Reply:
I do like the slickness of having it be transparently included in the bundle, but I can see how importing structured data and getting a string back is maybe not the best way for that to behave. Will change.

Reply:
As it turns out, doing `new URL("./default-clang-format.yaml", import.meta.url).toString()` evaluates to the file contents anyway? Slightly confused, but it does keep things simple.

Reply:
... and that's stopped working.
Switching to fetch.